crates/pool/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "kakarot-pool"
+version = "0.1.0"
+edition = "2021"
+
+[lints]
+workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+reth-transaction-pool = { workspace = true }
+reth-primitives = { workspace = true }
+reth-chainspec = { workspace = true }
+reth-node-ethereum = { workspace = true }
```

Comment:
I think you are missing a line break here

Reply:
fixed